disk-utils/sfdisk.c

```diff
@@ -1069,6 +1070,14 @@ static int command_dump(struct sfdisk *sf, int argc, char **argv)
 	if (!dp)
 		err(EXIT_FAILURE, _("failed to allocate dump struct"));
 
+	if (sf->no_device_names) {
+		rc = fdisk_script_set_no_device_names(dp, 1);
+		if (rc != 0 && !sf->quiet) {
+			fdisk_warnx(sf->cxt,
+				_("warning: libfdisk does not support --no-device-names, device names will be shown"));
```

Comment:
This message is incorrect. `libfdisk` does support it, but it failed.
As `sfdisk` and `libfdisk` are developed together we *know* that this error will never happen.
So IMO we either skip the return value check completely or just do `errx(EXIT_FAILURE, "set_no_device_names() failed");`